pele/gui/bhrunner.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, system, comm, nsteps=None):
         self.nsteps = nsteps
     
     def run(self):
-        seed = int(time.time() * 100.)
+        seed = int(time.time() * 100.) % 4294967295
```

Comment:
because it fails otherwise, that caps maximum value of the seed
On 3 Dec 2015 8:06 am, "Jacob Stevenson" <notifications@github.com> wrote:

> In pele/gui/bhrunner.py
> <https://github.com/pele-python/pele/pull/138#discussion_r46522482>:
>
> > @@ -19,7 +19,7 @@ def __init__(self, system, comm, nsteps=None):
> >          self.nsteps = nsteps
> >
> >      def run(self):
> > -        seed = int(time.time() * 100.)
> > +        seed = int(time.time() * 100.) % 4294967295
>
> why did you modify this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pele-python/pele/pull/138/files#r46522482>.
>